documentation/usage/errorcodes/index.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: documentation
+menu: errorcodes
+subtitle: Error Codes
+---
+
+# Error Codes
+
+When Flyway commands fail, they throw an exception with a message to help you identify the problem. They also contain an error code which users of the API or those who have enabled machine readable output can inspect and handle accordingly. Below are details of each error code under the command that causes it along with a suggested solution.
+
+
+## Validate Error Codes
+These error codes are surfaced when running `validate` or `validateWithResult`.
+
+### `SCHEMA_DOES_NOT_EXIST`
+- **Caused by:** The schema being validated against does not exist
+- **Solution:** Manually create the schema or enable `flyway.createSchemas`
```

Comment:
Could we turn all these references to config parameters into links to the relevant docs pages?